gamma_inc_test.go

```diff
@@ -2,11 +2,13 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package mathext
+package mathext_test
```

Comment:
I've seen it in a lot of third party libraries as well but perhaps they were influenced by the standard library as well. Reverting those changes now.